dictionary/README.md

```diff
@@ -1,5 +1,10 @@
 
 Please write a couple of sentences explaining your choice of restart
-strategy.
-
+strategy.  
 
+I chose `:transient` as the restart strategy, so that if the Dictionary
```

Comment:
Although I normally deduct a couple for transient (vs permanent) I agree with your scenario here.